.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,3 +4,5 @@ _PR changes summary to go here._
 
 - [ ] Tests added for new features
 - [ ] Test engineer approval
+
```

Comment:
```suggestion
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.
```

Reply:
I think the likelihood of someone merging a PR then scrolling up to tick this box is very low as people tend to merge, delete branch and then move on to the next page.

Reply:
Ah, it's intended to be ticked off _before_ merge (want to avoid merging PRs which don't have all checkboxes ticked). Is that not compatible with the merging documentation instructions?

Reply:
Ahhh that makes more sense, I'll update 😄